Resources/config/yui.yml

```diff
@@ -1039,3 +1040,5 @@ system:
                 ez-app-extension-admin:
                     requires: ['ez-serversideview', 'ez-serversideviewservice', 'ez-app-extension']
                     path: %ez_platformui.public_dir%/js/apps/extensions/ez-app-extension-admin.js
+                ez-translator:
```

Comment:
nitpick: can you move this block near the others *external* bridges in yui.yml ?